ouroboros-consensus-cardano/test/cardano-test/Main.hs

```diff
@@ -16,7 +17,7 @@ import           Test.Util.TestEnv (defaultMainWithTestEnv,
                      defaultTestEnvConfig)
 
 main :: IO ()
-main = do
+main = withStdTerminalHandles $ do
   hSetBuffering stdout LineBuffering
   hSetEncoding stdout utf8
   defaultMainWithTestEnv defaultTestEnvConfig tests
```

Comment:
You can minimize the diff a bit by adapting `defaultMainWithTestEnv` to call `withStdTerminalHandles`, instead of having to adapt evey single `test-suite`'s `main`.

Reply:
I'm not super convinced myself, I think it is nice to make it explicit in the main file rather than pushing it in a helper, but that is not a strong opinion. Done in 8a5bb42ffe236582fba64d84f4ee66c0d2640875.